__tests__/app.test.js

```diff
@@ -145,8 +145,52 @@ describe('/api/articles/:article_id', () => {
             expect(body.msg).toBe('Bad request')
         })
     })
+
+    test('PATCH 200: responds with the updated article and increments the given article_id.votes by amount given in object, ', () => {
+        const patchVotesObject = { inc_votes: -10}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(patchVotesObject)
+        .expect(200)
+        .then(({body: {updatedArticle}}) => {
+            expect(updatedArticle.votes).toBe(90)
+        })
+    })
+
+    test('PATCH 200: updates articles that did not have a votes property with new votes', () => {
+        const patchVotesObject = { inc_votes: 10}
+        return request(app)
+            .patch('/api/articles/3')
+            .send(patchVotesObject)
+            .expect(200)
+            .then(({body: {updatedArticle}}) => {
+                expect(updatedArticle.votes).toBe(10)
+        })
+    })
+
+    test('PATCH 400: responds with an 400 and msg when provided with incorrect type in object', () => {
+        const patchVotesObject = { inc_votes: "incorrect value type"}
+        return request(app)
+            .patch('/api/articles/3')
+            .send(patchVotesObject)
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toBe('Bad request')
+        })
+    })
+    test('PATCH 400: responds with an 400 and msg when provided with malformed object', () => {
+        const patchVotesObject = {}
+        return request(app)
+            .patch('/api/articles/3')
+            .send(patchVotesObject)
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toBe('Bad request')
+        })
```

Comment:
It's also worth considering what happens when we try to patch an article with an id that doesn't exist or an id that is invalid 👍